src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -643,7 +672,7 @@ def _set_backend(self, module_name):
     @_read_cli('orchestrator status',
                desc='Report configured backend and its status')
     def _status(self):
-        o = self._select_orchestrator()
+        o = self.orchestrator
```

Comment:
I've always disliked the way module options are implicitly added as attributes. Like e.g. it breaks type checking and it might create name-conflicts with elements on the class itself. Can we do 

```suggestion
        o = self.get_module_option("orchestrator")
```

instead?

Reply:
How does it relate to type checking?